usr/local/share/bastille/setup.sh

```diff
@@ -30,24 +30,28 @@
 
 bastille_config="/usr/local/etc/bastille/bastille.conf"
 . /usr/local/share/bastille/common.sh
+
+if [ ! -f ${bastille_config} ]; then
+    cp /usr/local/etc/bastille/bastille.conf.sample ${bastille_config}
+fi
```

Comment:
This actually has no effect, please use "" when checking the variable

like `if [ ! -f "${bastille_config}" ]; then`

Reply:
fixed, thank you